makefile

```
@@ -0,0 +1,8 @@
+migrations/litellm/sync:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)